src/mono-runtimes/ProfileAssemblies.projitems

```diff
@@ -298,8 +298,6 @@
     <MonoTestAssembly Include="monodroid_System_xunit-test.dll">
       <TestType>xunit</TestType>
     </MonoTestAssembly>
-    <MonoTestAssembly Include="System.Reflection.TestModule.dll">
```

Comment:
This isn't the right way to handle this:  this assembly is used by the `monodroid_corlib_xunit-test.dll` while testing reflection.